tests/unit/plugin/test_plugins.py

```
@@ -0,0 +1,1256 @@
+            ('schema_version: "1.0"\ninvalid_category: "not a dict"', "invalid category type"),
+            ('schema_version: "1.0"\ntest_category:\n  invalid_type: "not a dict"', "invalid type spec"),
+            ('schema_version: "1.0"\ntest_category:\n  missing_class:\n    description: "No class"', "missing class field"),
+            ('schema_version: "99.0"\ntest_category:\n  test_type:\n    class: tests.unit.plugin.test_plugins:DummyClass', "unsupported schema"),
+        ],
+    )  # fmt: skip
+    def test_load_manifest_warnings(
+        self,
+        empty_registry: _PluginRegistry,
+        tmp_path: Path,
+        yaml_content: str,
+        description: str,
```

Ruff (0.14.14)

[warning] Unused method argument: `description`

(ARG002)